### ISSUE TITLE
Unable to refresh OIDC .well-known or certificates

### ISSUE THREAD

**Issue by nl2412** [Peter-Slump/django-keycloak#47]
Hello!
Thanks for the great library. Unfortunately, I am currently having problem setting up and integrate it in my project. 
Every time I request refreshing OIDC .well-known or certificates from django admin console, I get this error:

```
ConnectionError at /admin/django_keycloak/realm/

HTTPSConnectionPool(host='localhost', port=8080): Max retries exceeded with url: /auth/realms/xxxxx/.well-known/openid-configuration (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f00ac0520f0>: Failed to establish a new connection: [Errno 111] Connection refused',))
```

My current project is using:
- Python 3.x
- Django 2.2.13
- Keycloak 10.0.2

All helps are appreciated! Thanks

**Comment by srinathganesh1** [comment]
- is it possible for you to provate an IP Address instead of "localhost"? and see whether that helps

- Cloud you share what url (localhost or what) is set in the page `/admin/django_keycloak/server/{ID}/change/`

**Comment by nl2412** [comment]
Ahh you are right my django app is running in a container so localhost won't work here. Thanks !

**Comment by mmcclelland1002** [comment]
> Ahh you are right my django app is running in a container so localhost won't work here. Thanks !

what was the fix? 0.0.0.0:8080 did not work for me nor the name of my docker compose server keycloak:8080. still getting the same error. Thanks!
